src/utils/isEquivalent.js

```diff
@@ -0,0 +1,21 @@
+export default function isEquivalent(a, b){
```

Comment:
give a and b more descriptive names to indicate what they are, and rename the function from isEquivalent to something like sameProps.

also it is suspicious to me that you would need to do this check.. generally you shouldn't need to check if two sets of properties are equivalent.